src/main/java/io/arlas/server/app/ArlasServer.java

```diff
@@ -122,11 +135,13 @@ public void run(ArlasServerConfiguration configuration, Environment environment)
         environment.getObjectMapper().configure(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, false);
         environment.jersey().register(MultiPartFeature.class);
         environment.jersey().register(new ArlasExceptionMapper());
+        environment.jersey().register(new WFSExceptionMapper());
```

Comment:
Since WFSException extends ArlasException, why it is not mapped in ArlasExceptionMapper with other children of ArlasException?

Reply:
It's a good question, thanks for asking